### ISSUE TITLE
1.4.2 version of the image installs jupyterhub 1.3.0

### ISSUE THREAD

**Issue by jmmk** [bitnami/bitnami-docker-jupyterhub#1]
Currently running `bitname/jupyterhub:1.4.2-debian-10-r81` which is the default from https://github.com/bitnami/charts/tree/master/bitnami/jupyterhub.

I noticed this in the logs when jupyterhub starts up:

```
[I 2021-11-04 20:36:20.318 JupyterHub app:2349] Running JupyterHub version 1.3.0                                                                                                                                          ...                                                                              │
[W 2021-11-04 20:36:20.473 JupyterHub _version:73] jupyterhub version 1.3.0 != jupyterhub-singleuser version 1.4.2. This could cause failure to authenticate and result in redirect loops!
```

And if I run `jupyterhub --version` with the image, it outputs 1.3.0
```
$ docker run docker.traackr.com/bitnami/jupyterhub:1.4.2-debian-10-r81 --version
1.3.0
```

Looking at `https://downloads.bitnami.com/files/stacksmith/jupyterhub-1.4.2-0-linux-amd64-debian-10.tar.gz`, which is downloaded in the Dockerfile, I can see in `files/miniconda/lib/python3.7/site-packages/jupyterhub/_version.py` the version 1.3.0, where in the 1.4.2 release (https://github.com/jupyterhub/jupyterhub/blob/909b3ad4d708a7369343c8a8537c95c05aae3b67/jupyterhub/_version.py) it is 1.4.2.

**Comment by pablogalegoc** [comment]
Thanks for reporting it @jmmk! I've opened an internal task to solve it

**Comment by pablogalegoc** [comment]
Hi again @jmmk! This has been fixed since `jupyterhub:1.5.0-debian-10-r7`. Thanks for taking the time to report it.
```
❯ docker run bitnami/jupyterhub:1.5.0-debian-10-r7 --version
1.5.0
```
